projects/aca-content/src/lib/components/files/files.component.ts

```diff
@@ -36,18 +36,25 @@ import { SetCurrentFolderAction, isAdmin, AppStore, UploadFileVersionAction, sho
 import { BreakpointObserver, Breakpoints } from '@angular/cdk/layout';
 import { debounceTime, takeUntil } from 'rxjs/operators';
 import { FilterSearch, ShareDataRow } from '@alfresco/adf-content-services';
-import { DocumentListPresetRef } from '@alfresco/adf-extensions';
+import { ContentActionRef, DocumentListPresetRef } from '@alfresco/adf-extensions';
 import { Observable } from 'rxjs';
 
 @Component({
-  templateUrl: './files.component.html'
+  templateUrl: './files.component.html',
+  styleUrls: ['./files.component.scss']
 })
 export class FilesComponent extends PageComponent implements OnInit, OnDestroy {
   isValidPath = true;
   isSmallScreen = false;
   isAdmin = false;
   selectedNode: MinimalNodeEntity;
   queryParams = null;
+  searchMode = true;
+  searchVisibility = false;
+  isMainActionPresent: boolean;
```

Comment:
you copied `isMainActionPresent` from `CreateMenuComponent` but this feature is not used in this component